packages/app/background-jobs-common/src/background-job-processor/processors/AbstractBackgroundJobProcessor.ts

```diff
@@ -85,11 +87,12 @@ export abstract class AbstractBackgroundJobProcessor<
 > {
   protected readonly logger: CommonLogger
 
-  private readonly redis: Redis
+  private redis?: Redis
```

Comment:
🟡 Please correct me if I am wrong, but I think we are not using this connection right? could we get rid of it?

Reply:
You're right, good catch! ❤️